src/main/io/serial.c

```diff
@@ -28,6 +28,8 @@
 #include "config/parameter_group.h"
 #include "config/parameter_group_ids.h"
 
+#include "config/config_master.h"
```

Comment:
Erk. That was me fixing a compile error, grabbing the first thing that fit. Replaced with `config.h`.